### PR TITLE
WELD-1794 First attempt to integrate Probe module

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: probe/core/src/main/java/org/jboss/weld/probe/ProbeServlet.java [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+            String contentType = detectContentType(resourceName);
+
+            if (IMG_PNG.equals(contentType)) {
+                if (!writeResource(resourceName, resp.getOutputStream())) {
+                    resp.sendError(HttpServletResponse.SC_NOT_FOUND);
+                }
+            } else {
+                // All other content types are text-based
+                String content = getResourceAsString(resourceName);
+                if (content == null) {
```
Comment (mkouba) [inline probe/core/src/main/java/org/jboss/weld/probe/ProbeServlet.java]: What do you mean? `getResourceAsString()` returns null if there is no resource found.

**Comment by weld-tester** [comment]
Triggering build using a merge of b6f57347dc419f73ef1f3651749a43790238b79c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 388 is now running using a merge of b6f57347dc419f73ef1f3651749a43790238b79c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/388

**Comment by weld-tester** [comment]
Build 388 outcome was SUCCESS using a merge of b6f57347dc419f73ef1f3651749a43790238b79c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/388
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/388

**Comment by weld-tester** [comment]
Triggering build using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 389 is now running using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Build 389 outcome was FAILURE using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Build 389 outcome was FAILURE using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Build 389 outcome was FAILURE using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Build 389 outcome was FAILURE using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Build 389 outcome was FAILURE using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Build 389 outcome was FAILURE using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Build 389 outcome was FAILURE using a merge of 3072afaef2e9225630ddd6dba5ccab985788f419 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/389
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/389

**Comment by weld-tester** [comment]
Triggering build using a merge of d154aa726b99baefb8dcf4089ec27e4e719aa513 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 398 outcome was FAILURE using a merge of d154aa726b99baefb8dcf4089ec27e4e719aa513 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/398
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/398

**Comment by weld-tester** [comment]
Build 398 outcome was FAILURE using a merge of d154aa726b99baefb8dcf4089ec27e4e719aa513 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/398
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/398

**Comment by weld-tester** [comment]
Build 398 outcome was FAILURE using a merge of d154aa726b99baefb8dcf4089ec27e4e719aa513 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/398
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/398

**Comment by weld-tester** [comment]
Build 398 outcome was FAILURE using a merge of d154aa726b99baefb8dcf4089ec27e4e719aa513 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/398
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/398

**Comment by weld-tester** [comment]
Build 398 outcome was FAILURE using a merge of d154aa726b99baefb8dcf4089ec27e4e719aa513 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/398
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/398

**Comment by weld-tester** [comment]
Build 398 outcome was FAILURE using a merge of d154aa726b99baefb8dcf4089ec27e4e719aa513 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/398
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/398

**Comment by weld-tester** [comment]
Triggering build using a merge of 0d69b5ddefa1735cf973bcb3497247a8581dfa1c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 401 outcome was FAILURE using a merge of 0d69b5ddefa1735cf973bcb3497247a8581dfa1c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/401
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/401

**Comment by weld-tester** [comment]
Build 401 outcome was FAILURE using a merge of 0d69b5ddefa1735cf973bcb3497247a8581dfa1c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/401
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/401

**Comment by weld-tester** [comment]
Triggering build using a merge of 6e6a8b80aab1f4a7844e8da4e2f167d157ac058a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 406 outcome was FAILURE using a merge of 6e6a8b80aab1f4a7844e8da4e2f167d157ac058a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/406
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/406

**Comment by weld-tester** [comment]
Build 406 outcome was FAILURE using a merge of 6e6a8b80aab1f4a7844e8da4e2f167d157ac058a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/406
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/406

**Comment by weld-tester** [comment]
Build 406 outcome was FAILURE using a merge of 6e6a8b80aab1f4a7844e8da4e2f167d157ac058a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/406
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/406

**Comment by weld-tester** [comment]
Build 406 outcome was FAILURE using a merge of 6e6a8b80aab1f4a7844e8da4e2f167d157ac058a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/406
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/406

**Comment by weld-tester** [comment]
Build 406 outcome was FAILURE using a merge of 6e6a8b80aab1f4a7844e8da4e2f167d157ac058a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/406
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/406

**Comment by mkouba** [comment]
@jharting The pull request is ready to merge :-)

**Comment by weld-tester** [comment]
Triggering build using a merge of 1b8a9e4996dca915786f38c5c8332cab59acab83 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 409 outcome was FAILURE using a merge of 1b8a9e4996dca915786f38c5c8332cab59acab83 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/409
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/409

**Comment by weld-tester** [comment]
Build 409 outcome was FAILURE using a merge of 1b8a9e4996dca915786f38c5c8332cab59acab83 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/409
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/409

**Comment by weld-tester** [comment]
Build 409 outcome was FAILURE using a merge of 1b8a9e4996dca915786f38c5c8332cab59acab83 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/409
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/409

**Comment by weld-tester** [comment]
Build 409 outcome was FAILURE using a merge of 1b8a9e4996dca915786f38c5c8332cab59acab83 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/409
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/409

**Comment by weld-tester** [comment]
Triggering build using a merge of 8607acfdf50741bab01c248ee840e849f90d5528 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 415 is now running using a merge of 8607acfdf50741bab01c248ee840e849f90d5528 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/415

**Comment by weld-tester** [comment]
Build 415 outcome was FAILURE using a merge of 8607acfdf50741bab01c248ee840e849f90d5528 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/415
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/415

**Comment by weld-tester** [comment]
Build 415 outcome was FAILURE using a merge of 8607acfdf50741bab01c248ee840e849f90d5528 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/415
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/415

**Comment by weld-tester** [comment]
Build 415 outcome was FAILURE using a merge of 8607acfdf50741bab01c248ee840e849f90d5528 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/415
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/415

**Comment by weld-tester** [comment]
Build 415 outcome was FAILURE using a merge of 8607acfdf50741bab01c248ee840e849f90d5528 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/415
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/415

**Comment by weld-tester** [comment]
Triggering build using a merge of 7f9bc7c012f3a4d630dfad91e1b4fa52421b23f9 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 417 is now running using a merge of 7f9bc7c012f3a4d630dfad91e1b4fa52421b23f9 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/417

**Comment by weld-tester** [comment]
Build 417 outcome was SUCCESS using a merge of 7f9bc7c012f3a4d630dfad91e1b4fa52421b23f9 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/417
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/417

**Comment by jharting** [comment]
Merged, thanks!

**Pull request merged**